src/components/case/CreateForm.vue

```diff
@@ -194,11 +224,20 @@ export default {
       };
       this.tags.push(newTag);
       this.newTag = "";
-      this.tagadded = true;
+      this.tagAdded = true;
     },
     removeTag(index) {
       this.tags.splice(index, 1);
-      this.tagadded = false;
+      this.tagAdded = false;
+    },
+
+    addFile(files) {
+      this.files.push(files);
```

Comment:
Please remove `console.log()` apart from `.catch()` blocks. This is applicable for all such occurrences in this PR.